config/projects.yaml

```diff
@@ -233,7 +233,7 @@ doctrine-phpcr-admin-bundle:
     custom_doctor_rst_whitelist_part:
         "    - '.. _`cmf-sandbox configuration`: https://github.com/symfony-cmf/cmf-sandbox/blob/master/app/config/config.yml'"
     excluded_files:
-        - phpunit.xml.dist.twig
+        - tests/bootstrap.php.twig
```

Comment:
tests or test? 

The title does not match the change

Reply:
Updated